scripts/pre-commit.mjs

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Pre-commit hooks are not firing. This file will fail when hooks fire.

Reply:
Jason and I are pair programming now and are struggling to figure out why the pre-commit hooks are failing. Would you have an idea why?

Reply:
Pre-commit hooks are now firing, nothing happened with this file.